lib/Onyx.js

```diff
@@ -1684,6 +1696,7 @@ const Onyx = {
     clear,
     getAllKeys,
     init,
+    isUsingFallbackStorageProvider,
```

Comment:
Similarly, to having the return value for `Onyx.init()` answer this question - not sure who needs to know this.

Reply:
Maybe it would be useful to return the provider name vs. a boolean and then the client code could decide if they need this or not.

Reply:
We can also do that, definitely. Just thought it might make sense to assume that the user/developer who uses Onyx isn't aware of all the internal providers and therefore only wants to know if it's working 100% or not.

We could also change this function to something like `getOnyxStats` or sth similar, which gives all infos relevant to performance/etc...

Reply:
I think that gets me closer to having this just say what it means. Instead of alluding to a "fallback storage provider" maybe something that clearly indicates the "user can't persist data because storage doesn't work".

Reply:
I think Marc is getting close now to what the frontend developer wants to know. They don't want to know the storage provider being used, or if it's memory only, they only really care that the data is being persisted to disk or not (since the whole point of Onyx is to persist the state to disk). I also agree there should only be one way to check for this. 

However, when I look over my previous comment suggesting the application could show a banner that performance is degraded, I fear I was pre-optimizing.

Let's keep it simple for now and only build what we need. We have no plans to display any kind of banner like I suggested, so let's not try to build in support for it now. I think we can remove:

1. The return value from `Onyx.init()`
2. The exported method `isUsingFallbackStorageProvider()`